src/Symfony/Component/CssSelector/Node/SpecificityAdjustmentNode.php

```diff
@@ -34,7 +34,7 @@ public function __construct(
 
     public function getSpecificity(): Specificity
     {
-        return new Specificity(0, 0, 0);
+        return $this->selector->getSpecificity()->plus(new Specificity(0, 0, 0));
```

Comment:
this could be simplified as `return $this->selector->getSpecificity();`

Reply:
Indeed !